core/base/inc/ROOT/TSequentialExecutor.hxx

```diff
@@ -13,96 +13,114 @@
+      /// \return The number of workers in the pool, one.
+      unsigned GetPoolSize() { return 1u; }
    };
 
    /************ TEMPLATE METHODS IMPLEMENTATION ******************/
 
    //////////////////////////////////////////////////////////////////////////
-   /// Execute func (with no arguments) nTimes.
-   /// Functions that take more than zero arguments can be executed (with
-   /// fixed arguments) by wrapping them in a lambda or with std::bind.
+   /// \brief Execute a function without arguments several times, dividing the execution in nChunks.
+   ///
+   /// \param func Function to be executed.
+   /// \param nTimes Number of times function should be called.
    template<class F>
    void TSequentialExecutor::Foreach(F func, unsigned nTimes) {
       for (auto i = 0U; i < nTimes; ++i) func();
    }
 
    //////////////////////////////////////////////////////////////////////////
-   /// Execute func, taking an element of a
-   /// sequence as argument.
+   /// \brief Execute a function over a sequence of indexes, dividing the execution in nChunks.
+   ///
+   /// \param func Function to be executed. Must take an element of the sequence passed assecond argument as a parameter.
+   /// \param args Sequence of indexes to execute `func` on.
    template<class F, class INTEGER>
    void TSequentialExecutor::Foreach(F func, ROOT::TSeq<INTEGER> args) {
-       for(auto i : args) func(i);
+         for(auto i : args) func(i);
    }
 
-   /// \cond
    //////////////////////////////////////////////////////////////////////////
-   /// Execute func, taking an element of a
-   /// initializer_list as argument.
+   /// \brief Execute a function over the elements of an initializer_list, dividing the execution in nChunks.
+   ///
+   /// \param func Function to be executed on the elements of the initializer_list passed as second parameter.
+   /// \param args initializer_list for a vector to apply `func` on.
    template<class F, class T>
    void TSequentialExecutor::Foreach(F func, std::initializer_list<T> args) {
-       std::vector<T> vargs(std::move(args));
-       Foreach(func, vargs);
+         std::vector<T> vargs(std::move(args));
+         Foreach(func, vargs);
    }
-   /// \endcond
 
    //////////////////////////////////////////////////////////////////////////
-   /// Execute func, taking an element of an
-   /// std::vector as argument.
+   /// \brief Execute a function over the elements of a vector, dividing the execution in nChunks.
+   ///
+   /// \param func Function to be executed on the elements of the vector passed as second parameter.
+   /// \param args Vector of elements passed as an argument to `func`.
    template<class F, class T>
    void TSequentialExecutor::Foreach(F func, std::vector<T> &args) {
-        unsigned int nToProcess = args.size();
-        for(auto i: ROOT::TSeqI(nToProcess)) func(args[i]);
+         unsigned int nToProcess = args.size();
+         for(auto i: ROOT::TSeqI(nToProcess)) func(args[i]);
+   }
```

Comment:
Indentation seems to be inconsistent for the file. Should be a multiple of 3, there's a clang-format config file that might be of help.

Reply:
I realized that, but I wanted to adress it in another PR (both for this file and TThreadExecutor) to avoid all the noise